.azure/obtemplates/build-linux-packages.yml

```diff
@@ -46,7 +46,6 @@ jobs:
   - ${{ if not(eq(parameters.sign, false)) }}:
     - task: onebranch.pipeline.signing@1
       displayName: Sign
-      target: linux_build_container2
```

Comment:
Why did you remove this?

Reply:
I will use the linux_build_container be default. Now that both linux_build_container and linux_build_container2 are cross-compile build dockers, it doesn't matter which we use.